packages/clerk-js/src/v4/styledSystem/sizes.ts

```diff
@@ -0,0 +1,54 @@
+const space = {
+  px: '1px',
+  '0x5': '0.125rem',
```

Comment:
I dynamically prefix all keys with `$` so we can freely choose key names without weird accessing patterns, even if they start with a number:
```ts
paddingBottom: theme.sizes['2xl']
```

For keys containing decimal values, I suggest we follow the `0x5` (0,5) convention because intellisense just works with this format: 
```ts
// this is ok
paddingBottom: theme.sizes.$0x5
```